src/components/Settings/index.tsx

```diff
@@ -21,11 +23,15 @@ const interfaceLanguageList = [
     },
     {
         name: 'Medžuslovjansky',
-        value: 'isv-Latn',
+        value: 'isv-latn',
```

Comment:
@scherebedov , 

>  An implementation can reproduce this format without accessing the
   registry as follows.  All subtags, including extension and private
   use subtags, use lowercase letters with two exceptions: two-letter
   and four-letter subtags that neither appear at the start of the tag
   nor occur after singletons.  Such two-letter subtags are all
   uppercase (as in the tags "en-CA-x-ca" or "sgn-BE-FR") and four-
   letter subtags are titlecase (as in the tag "az-Latn-x-latn").

**are titlecase**.

Reply:
Fixed